.gitignore

```diff
@@ -1,3 +1,4 @@
-__pycache__
-.vscode
-tools
\ No newline at end of file
+.vscode/
+.venv/
+__pycache__/
+tools/
```

Comment:
Hey, what do you have in tools? Anything nice? :)

Reply:
> Hey, what do you have in tools? Anything nice? :)

Oh no you caught me! That is just where I have a couple janky scripts to scrape the data for the airport.hjson file. You didnt think I did that on my own xD

Reply:
I _hoped_ you didn't! lol